scss/_variables.scss

```diff
@@ -339,6 +339,7 @@ $border-radius:               .25rem !default;
 $border-radius-sm:            .2rem !default;
 $border-radius-lg:            .3rem !default;
 
+$rounded-sizes:               map-remove($spacers, 4, 5) !default;
```

Comment:
I think we better redelare the sizes here, would be weird if the spacers map is changed to :
```scss
$spacers: (
  0: 0,
  1: $spacer / 4,
  2: $spacer / 2,
  3: $spacer,
  4: $spacer * 1.5,
  5: $spacer * 3,
  6: $spacer * 4.5,
) !default;
```